parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -316,7 +331,13 @@ static Context dataContext(Map<String, String> config) {
 
         String compressionLevel = config.getOrDefault(PARQUET_COMPRESSION_LEVEL, PARQUET_COMPRESSION_LEVEL_DEFAULT);
 
-        return new Context(rowGroupSize, pageSize, dictionaryPageSize, codec, compressionLevel);
+        int rowGroupCheckMinRecordCount = Integer.parseInt(config.getOrDefault(
```

Comment:
can use `PropertyUtil.propertyAsInt`

Reply:
Originally, I was trying to stay with the existing code/style in this class. In the new commit, I have updated `Integer.partInt` to `PropertyUtil.propertyAsInt` for all integer configs (old or new)